nullaway/build.gradle

```diff
@@ -47,6 +57,10 @@ dependencies {
     epJavac deps.build.checkerDataflow
 }
 
+shadowJar {
+    classifier = "cf"
```

Comment:
Our NullAway+CF jar will be called `nullaway-version-cf.jar`

For now, I am excluding most dependencies, only bundling CF-dataflow and CF-javacutil. But let me know if we want to make a true fat jar, with EP and javax and everything.

Reply:
No, this is right, we just want to bundle the CF classes

Reply:
is the classifier here a maven classifier?

Reply:
I am not sure, it gets added at the end of that jar filename, I am not sure were else it is used. I'd be fine with setting it to `null` and having both `shadow` be our default build and this being our default jar.